.github/workflows/update-bpmn-visualization-version.yml

```
@@ -15,33 +15,33 @@ jobs:
       VERSION: ${{ github.event.client_payload.version || github.event.inputs.version }}
     steps:
       - uses: actions/checkout@v3
-      - name: Get old bpmn-visualization version
+      - name: Get the old version of the bpmn-visualization TypeScript library
```

This is too verbose, the name of the workflow is explicit enough.
```suggestion
      - name: Get the old version of bpmn-visualization
```